assets/samplesheet.csv

```diff
@@ -1,5 +1,5 @@
 sample,mlst_alleles,address
-sampleQ,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sampleQ.mlst.json,
-sample1,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sample1.mlst.json,1.1.1
-sample2,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sample2.mlst.json,1.1.1
-sample3,https://raw.githubusercontent.com/phac-nml/gasnomenclature/dev/tests/data/reports/sample3.mlst.json,1.1.2
+sampleQ,https://raw.githubusercontent.com/phac-nml/gasnomenclature/update/input_assure/tests/data/reports/sampleQ.mlst.json,
```

Comment:
Can you update the branch here for test data back to `dev`?

Reply:
Thank you for the find! Changed here: [03b0d3a](https://github.com/phac-nml/gasnomenclature/pull/21/commits/03b0d3a4a2257ec32dbaf69ab4588bf4fe9c3c2d)